scripts/tests.sh

```diff
@@ -30,6 +30,20 @@ set -e
 
 REPO_ROOT="$(dirname "$0")"/..
 
+if [ "$1" -e --junit_report ]
+then
+    if [ -z "$2" ]
+    then
+        echo "Usage: $0 [--junit_report <report_directory>]"
+        exit 1
+    fi
+    testargs_no_opt="--logger=JUNIT,test_suite,$2/no_opt.xml"
+    testargs_opt="--logger=JUNIT,test_suite,$2/opt.xml"
+else
+    testargs_no_opt=''
+    testargs_opt=''
```

Comment:
Any reason these are using single quotes as opposed to double quotes for the rest?

Reply:
No particular.